### DISCUSSION TITLE
`AxisControl` behaviour on changing from none to sync

### DISCUSSION THREAD

**Discussion by Fybex** [maxi-blocks/maxi-blocks#4101] [Q&A]
https://user-images.githubusercontent.com/46112160/208686225-2f4cba14-5d0f-4740-aa11-62dfc69e7887.mp4

So, as you can see from the video, when I set the value for top (none sync) and then changed the sync to axis, the value for bottom didn't appear, and the user sees the wrong value (top and bottom are 280px, but only top is 280px). IMO, on sync change, bottom value should be added. But what should happen on sync change if, for example, the top and bottom have different values? I decided to ask first before creating a problem because I'm not sure what behaviour is expected.

**Comment by Naaaaiix** [comment]
Thanks for such an interesting situation @Fybex; I think it's a complete UX issue we need to solve not just on position, also on Margin, Padding... anything under AxisControl. And my answer will be the same we always do:

how other plugins deal with it? lol
